src/main/scala/com/fulcrumgenomics/bam/EstimatePoolingFractions.scala

```diff
@@ -190,7 +189,7 @@ class EstimatePoolingFractions
     iterator.setIncludeNonPfReads(false)
     iterator.setMappingQualityScoreCutoff(this.minMappingQuality)
     iterator.setQualityScoreCutoff(this.minBaseQuality)
-    iterator.toIterator
+    javaIteratorAsScalaIterator(iterator)
   }
```

Comment:
It's interesting to see that we construct SamLocusIterators in a similar fashion across a lot of code.  It's starting to make me think it needs a proper treatment in the `Bam` object, but that's for another day.